src/io.jl

```diff
@@ -111,7 +111,7 @@ function dict2imas(dct::AbstractDict, @nospecialize(ids::T); error_on_missing_co
     return ids
 end
 
-function dict2imas(
+function dict2imas_original(
```

Comment:
Please remove `dict2imas_original`. We already have a lot of code to maintain, no need to keep unused code around :)